StudentList/Common/Extensions/ActivityExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Threading.Tasks;
+using Android.App;
+using StudentList.Common.Dialogs;
+
+namespace StudentList.Extensions
+{
+    public static class ActivityExtensions
+    {
+        public static async Task RunMethodWithLoaderAsync(this Activity activity, Task task)
```

Comment:
You handle Task, and how about Task<T>?

Reply:
![image](https://user-images.githubusercontent.com/26973559/42621819-7fbbc090-85c7-11e8-8c44-88b311180094.png)
What do you think about that?

Reply:
I think you will soon get tired of writing a new method for every task result (List<Student>, Student, int, double, etc). Maybe you can simply use generic for that?)